package.json

```diff
@@ -1,16 +1,15 @@
 {
   "license": "UNLICENSED",
   "devDependencies": {
-    "@outreach/semantic-release-circleci-orb": "^1.1.9",
```

Comment:
Need to add this orb back

Reply:
Make sure to run `yarn` after as well

Reply:
should we update stencil-base for a custom block?

Reply:
Yes, but `package.json` doesn't support blocks. I was going to write merging logic but still haven't got around to it :/